code/datums/spells/summonitem.dm

```diff
@@ -12,7 +12,8 @@
 	include_user = 1
 
 	var/obj/marked_item
-
+	/// List of objects which will result in the spell stopping with the recursion search
+	var/static/list/blacklisted_summons = list(/obj/machinery/computer/cryopod, /obj/machinery/atmospherics, /obj/structure/disposalholder, /obj/machinery/disposal)
```

Comment:
Can you make this use a proper typecache instead of just a regular type list?

Reply:
sure thing